.editorconfig

```diff
@@ -0,0 +1,4 @@
+# 2 space indentation
+[*.py]
+indent_style = space
+indent_size = 2
```

Comment:
See comment on #408, [I'm not sure why this file is needed](https://github.com/okTurtles/group-income-simple/pull/408/files#r189444991).